### PR TITLE
recommend against password protecting keys

### LINKED ISSUE #96: recommend against passwords for the OpenPGP key
We talked about this a bunch but have not included it in the spec yet.

### PR DESCRIPTION
**Pull request by azul**
(with @Valodim)

fixes #96

### CODE REVIEW

**Comment by Valodim** [comment]
:+1:

File: doc/level1.rst [hunk 1]
```diff
@@ -417,6 +417,12 @@ public keys.  It MAY support other OpenPGP key formats found in
 a ``type=1`` Autocrypt header (for example, by passing it agnostically
 to an OpenPGP backend for handling).
 
+We recommend against protecting the private key with a password. All
```
Comment (hpk42) [inline doc/level1.rst]: i'd like to be specific: "MUAs SHOULD NOT protect the secret Autocrypt key with a password."

File: doc/level1.rst [hunk 2]
```diff
@@ -417,6 +417,12 @@ public keys.  It MAY support other OpenPGP key formats found in
...
+encrypted outgoing messages MUST also be signed, which would require the
+user to enter their password for both reading and sending mail. This
+introduces too much friction to become part of a routine daily workflow.
+Protection of the user's keys at rest is achieved more easily and
+securely with full-disk encryption.
```
Comment (hpk42) [inline doc/level1.rst]: i'd add ", which also protects all kinds of other user data. Moreover, some architectures (e.g. mobile) already protect from other programs reading a mail app's data".

File: doc/level1.rst (hunk 1 above)
Comment (hpk42) [inline doc/level1.rst]: maybe add a subsection title of "passphrase-less secret keys" to make this something we can link to easily.  I consider this a major usability issue, not just a recommendation on the side.

File: doc/level1.rst (hunk 2 above)
Comment (Valodim) [inline doc/level1.rst]: I wouldn't add the second part. While true, it weakens our point for architectures that *don't* have that property. I like the first idea though, how about just adding "and other files"?:

"Protection of the user's keys and other files at rest is achieved more easily and securely with full-disk encryption."

**Pull request merged**